### ISSUE TITLE
After adding in Rout.php all artsian command or disable

### ISSUE THREAD

**Issue by miladganjali** [Zizaco/entrust#643]
When i add this line code in Rout.php

`Entrust::routeNeedsPermission('admin/post*', 'create-post');`

all response to artsian command become this

`php artisan vendor:publish

  [BadMethodCallException]
  Method filter does not exist.

`

**Comment by AmrAbdalrahman** [comment]
@miladganjali  Are you solve this issue?

**Comment by miladganjali** [comment]
@AmrAlmagic No, Entrust has many issues when you want to be used in the complex project, after this error and no response I stop using this package and create my own you can use a source code of "entrust" for inspiration.
